include/jemalloc/internal/pa.h

```diff
@@ -7,46 +7,17 @@
 #include "jemalloc/internal/edata_cache.h"
 #include "jemalloc/internal/emap.h"
 #include "jemalloc/internal/lockedint.h"
-
-enum pa_decay_purge_setting_e {
-	PA_DECAY_PURGE_ALWAYS,
-	PA_DECAY_PURGE_NEVER,
-	PA_DECAY_PURGE_ON_EPOCH_ADVANCE
-};
-typedef enum pa_decay_purge_setting_e pa_decay_purge_setting_t;
+#include "jemalloc/internal/pac.h"
+#include "jemalloc/internal/pai.h"
```

Comment:
Not sure if I'm understanding it correctly, but at the end of the refactoring `pa` should not need `pai` in theory, right? `pa` only depends on each `pai` implementation it needs, and the implementations each include `pai`?